x-pack/plugin/security/qa/basic-enable-security/src/javaRestTest/java/org/elasticsearch/xpack/security/EnableSecurityOnBasicLicenseIT.java

```diff
@@ -85,6 +97,21 @@ public void testSecuritySetup() throws Exception {
         }
     }
 
+    public void testSecurityDisabledWarning() throws Exception {
+        final Request request = new Request("GET", "/_cat/indices");
+        Response response = client().performRequest(request);
+        List<String> warningHeaders = response.getWarnings();
+        if (securityEnabled) {
+            assertThat (warningHeaders.isEmpty(), equalTo(true));
```

Comment:
```suggestion
            assertThat(warningHeaders, Matchers.empty());
```

Using the more explicit matchers means we get more meaningful error messages when they fail.
Instead of a simple 

> expected <true> but was <false>

(which tells us almost nothing), we'd get

> Expected: an empty collection
     but: <[foo]>